app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,7 @@
         android:requestLegacyExternalStorage="true">
         <activity
             android:name="com.omen273.crossLingo.GameActivity"
-            android:windowSoftInputMode="adjustResize"
+            android:windowSoftInputMode="stateAlwaysHidden|adjustResize"
```

Comment:
why was "stateAlwaysHidden" added?

Reply:
Because soft keyboard should never be shown